### PR TITLE
Upgrade to jbehave 5.0.0-alpha.11

### PR DESCRIPTION
**Pull request by uarlouski**

### CODE REVIEW

File: vividus-plugin-accessibility/src/main/java/org/vividus/bdd/converter/ParametersToAccessibilityCheckOptionsConverter.java [hunk 1]
```diff
@@ -43,44 +42,37 @@
...
-    private AccessibilityCheckOptions toOptions(Parameters row)
+    public AccessibilityCheckOptions convertValue(Parameters row, Type type)
     {
         AccessibilityStandard standardParameter = row.valueAs(STANDARD, AccessibilityStandard.class, null);
         checkNotNull(standardParameter, STANDARD);
         ViolationLevel level = row.valueAs(LEVEL, ViolationLevel.class, null);
         checkNotNull(level, LEVEL);
-        Set<Locator> locators = row.valueAs("elementsToCheck", new TypeLiteral<Set<Locator>>() { }.getType(), Set.of());
+        Set<Locator> locators = row.valueAs("elementsToCheck", new TypeLiteral<Set<Locator>>() { }.getType(),
+                Set.of());
```
Comment (valfirst) [inline vividus-plugin-accessibility/src/main/java/org/vividus/bdd/converter/ParametersToAccessibilityCheckOptionsConverter.java:75]: noisy change

File: vividus-plugin-email/src/main/java/org/vividus/bdd/email/converter/StringToListOfMessagePredicatesConverter.java [hunk 2]
```diff
@@ -17,40 +17,26 @@
...
 
 import org.apache.commons.lang3.function.FailablePredicate;
-import org.jbehave.core.model.ExamplesTable;
-import org.jbehave.core.steps.ParameterConverters.AbstractChainableParameterConverter;
+import org.jbehave.core.steps.ParameterConverters.AbstractParameterConverter;
 import org.jbehave.core.steps.Parameters;
 import org.vividus.bdd.email.factory.EmailParameterFilterFactory;
 
 @Named
 public class StringToListOfMessagePredicatesConverter
```
Comment (valfirst) [inline vividus-plugin-email/src/main/java/org/vividus/bdd/email/converter/StringToListOfMessagePredicatesConverter.java:31]: `ParametersToMessagePredicateConverter`

Comment (valfirst) [reply vividus-plugin-email/src/main/java/org/vividus/bdd/email/converter/StringToListOfMessagePredicatesConverter.java:31]: and do not forget to rename unit tests class

File: vividus-plugin-web-app/src/main/java/org/vividus/bdd/converter/ui/web/StringToListSequenceActionConverter.java [hunk 3]
```diff
@@ -34,8 +31,7 @@
 import org.vividus.bdd.steps.ui.web.model.SequenceActionType;
 
 @Named
-public class StringToListSequenceActionConverter
-        extends AbstractChainableParameterConverter<ExamplesTable, List<SequenceAction>>
+public class StringToListSequenceActionConverter extends AbstractParameterConverter<Parameters, SequenceAction>
```
Comment (valfirst) [inline vividus-plugin-web-app/src/main/java/org/vividus/bdd/converter/ui/web/StringToListSequenceActionConverter.java:34]: `ParametersToSequenceActionConverter`

File: vividus/src/main/java/org/vividus/bdd/converter/ParametersToListOfMapsConverter.java [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+import java.lang.reflect.Type;
+import java.util.Map;
+
+import javax.inject.Named;
+
+import org.jbehave.core.steps.ParameterConverters.AbstractParameterConverter;
+import org.jbehave.core.steps.Parameters;
+
+@Named
+public class ParametersToListOfMapsConverter extends AbstractParameterConverter<Parameters, Map<String, String>>
```
Comment (valfirst) [inline vividus/src/main/java/org/vividus/bdd/converter/ParametersToListOfMapsConverter.java:28]: `ParametersToMapConverter `

**Review by valfirst — CHANGES_REQUESTED** [review]

File: vividus-plugin-accessibility/src/main/java/org/vividus/bdd/converter/ParametersToAccessibilityCheckOptionsConverter.java (hunk 1 above)
Comment (uarlouski) [reply vividus-plugin-accessibility/src/main/java/org/vividus/bdd/converter/ParametersToAccessibilityCheckOptionsConverter.java:75]: 121

**Review by valfirst — APPROVED** [review]

**Pull request merged**